bin/auth.js

```diff
@@ -7,10 +7,9 @@ const config = require("./config"),
   lti = require("ims-lti"),
   RedisNonceStore = require("../node_modules/ims-lti/lib/redis-nonce-store.js"),
   redis = require("redis"),
-  redis_client =
-    process.env.NODE_ENV === "development"
-      ? require("redis").createClient()
-      : require("redis").createClient(config.redisURL),
+  redis_client = process.env.REDIS_URL
```

Comment:
Unless you're assuming that locally we're going to remove the redis url, I think there's an issue with this cause otherwise, it's always going to evaluate to true. Also I think the conditional is backwards, you would only want to `createClient` when there does exist a redis url.